myhpi/static/css/myHPI.css

```diff
@@ -203,4 +203,39 @@ nav .nav-link[aria-expanded=true] {
 h1.toc-title::after {
     margin: 0;
     height: 0px;
-}
\ No newline at end of file
+}
+
+.print-shown {
+    display: none;
+}
+
+@media print {
+    .print-none {
+        display: none !important;
+    }
+
+    .print-shown {
+        display: block !important;
+    }

```

Comment:
Boostrap already provides these utilities for us :)

Reply:
Yes, but they do not work for some reason for me.

Reply:
Ah they don't work because bootstrap is only loaded for media=screen, which can of course be changed